src/hooks/use-enhanced-transcript.ts

```diff
@@ -0,0 +1,8 @@
+import fetcher from 'utils/fetcher'
+import useSWR from 'swr'
+
+export const useEnhancedTranscript = (url: string) => {
+  const {data: transcriptData} = useSWR(url, fetcher)
+  const enhancedTranscript = transcriptData?.text
+  return enhancedTranscript
```

Comment:
@DrShpongle I removed the `{}` from this, no need to return an object if we are only interested in the transcript. I think technically this could just return `data?.text`